CP2077 - EasyInstall/Form1.cs

```diff
@@ -366,5 +378,63 @@ private void btnUninstall_Click(object sender, EventArgs e)
                 MetroFramework.MetroMessageBox.Show(this, "Uninstall was not able to delete the mod. Just delete <cyberpunk install path>/bin/x64/version.dll and <cyberpunk install path>/bin/x64/plugins/", "Exception!", MessageBoxButtons.OK, MessageBoxIcon.Error);
             }
         }
+        private void btnFindSteam_Click(object sender, EventArgs e)
+        {
+            try
+            {
+                string path = SteamGamePath.FindGameByAppID("1091500");
+                if (path == null)
+                {
+                    MetroFramework.MetroMessageBox.Show(this, "Error: Couldn't Find CyberPunk for Steam!", "File not found Error!", MessageBoxButtons.OK, MessageBoxIcon.Error);
+                    //MessageBox.Show("Error: Couldn't Find CyberPunk for steam!");
+                    return;
+                }
+                DialogResult result = MessageBox.Show(path, "Is this Correct?", MessageBoxButtons.YesNo);
+                if (result == DialogResult.Yes)
+                    PatchGame(path);
+                else if (result == DialogResult.No)
+                    MetroFramework.MetroMessageBox.Show(this, null, "Install Canceled.", MessageBoxButtons.OK);
+                else
+                {
+                    MetroFramework.MetroMessageBox.Show(this, "Error during installation\nError code: 2", "Critical Error!", MessageBoxButtons.OK, MessageBoxIcon.Error);
+                    btnMain.Text = "Critical Error!";
+                }
+            }
+            catch (Exception ex)
+            {
+                MetroFramework.MetroMessageBox.Show(this, "Error: " + ex, "Unknown Error!", MessageBoxButtons.OK, MessageBoxIcon.Error);
+                //MessageBox.Show("Error" + ex);
+            }
+        }
+
+        private void btnFindGoG_Click(object sender, EventArgs e)
+        {
+            try
+            {
+                string path = GoGGamePath.FindGameByAppID("1423049311");
+                if (path == null)
+                {
+                    MetroFramework.MetroMessageBox.Show(this, "Error: Couldn't Find CyberPunk for GoG!", "File not found Error!", MessageBoxButtons.OK, MessageBoxIcon.Error);
+                    //MessageBox.Show("Error: Couldn't Find CyberPunk for GoG!");
+                    return;
+                }
+                //DialogResult dialogResult = MessageBox.Show(path, "Is this Correct?", MessageBoxButtons.YesNo);
+                DialogResult result = MetroFramework.MetroMessageBox.Show(this, path, "Is this Correct?", MessageBoxButtons.YesNo);
+                if (result == DialogResult.Yes)
+                    PatchGame(path);
```

Comment:
```suggestion
                    PatchGame($"{path}\\bin\\x64");
```
This will work, but it depends on where you want to append the additional parts of the path.

Reply:
Ahh my bad. I assumed PatchGame added "bin\x64" to it. Not that you had to pass the x64 path